### PR TITLE
Payable runSQL, URITemplate util, lockController method, and other things

### PR DESCRIPTION
**Pull request by sanderpick**
- Adds `payable` to `runSQL` and `ITablelandController.getPolicy`
- Adds `URITemplate` util
- Publishes all contracts to NPM
- Adds contract verification script (we can verify once we upgrade to this version of `TablelandTables`)
- Removes `ERC721ABurnable`, which is not useful for tables
- Adds a new storage mapping for locked table controllers. A table controller can be locked by the owner by calling `lockController`. A locked controller cannot be set or re-locked.
- Removes maintainers from README and package.json. They redundant with the actual contributors shown natively by GH and just add more maintenance overhead.
- Renames repo to `evm-tableland`. Once this is merged, we can rename the repo and publish the re-named package to NPM.
- Updates deps

### CODE REVIEW

File: contracts/TablelandTables.sol [hunk 1]
```diff
@@ -100,12 +103,14 @@ contract TablelandTables is
...
     function _getPolicy(address caller, uint256 tableId)
         private
-        view
         returns (ITablelandController.Policy memory)
     {
         address controller = _controllers[tableId];
         if (_isContract(controller)) {
-            return ITablelandController(controller).getPolicy(caller);
+            return
+                ITablelandController(controller).getPolicy{value: msg.value}(
```
Comment (sanderpick) [inline contracts/TablelandTables.sol:111]: Pass on ether to controller contract.

File: contracts/test/TestTablelandController.sol [hunk 2]
```diff
@@ -3,20 +3,29 @@ pragma solidity ^0.8.4;
...
     function getPolicy(address caller)
         public
-        view
+        payable
         override
         returns (ITablelandController.Policy memory)
     {
+        // Enforce some ether and revert if insufficient
+        if (msg.value != 1 ether) {
+            revert InsufficientValue(msg.value, REQUIRED_VALUE);
```
Comment (sanderpick) [inline contracts/test/TestTablelandController.sol:26]: Example showing how a controller can gate runSQL with a fee 🤑

File: contracts/utils/SStrings.sol [hunk 3]
```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.4 <0.9.0;
+
+/**
+ * @dev This is a subset of string slice methods from https://github.com/Arachnid/solidity-stringutils.
```
Comment (sanderpick) [inline contracts/utils/SStrings.sol:5]: Some methods from a Strings lib that are used by `URITemplate`

File: contracts/utils/URITemplate.sol [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+import "./SStrings.sol";
+
+/**
+ * @dev Helper contract for constructing token URIs where the tokenId may not
+ * be at the end of the token URI, e.g., "https://foo.xyz/{id}?bar=baz".
+ *
+ * This is especially useful when driving token metadata from a Tableland query
+ * where tokenId may be embedded in the middle of the query string.
+ */
+contract URITemplate {
```
Comment (sanderpick) [inline contracts/utils/URITemplate.sol:13]: The URI Template helper we're using in Rigs.

File: scripts/verify.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  if (proxy === "") {
+    throw Error(`no proxy entry for '${network.name}'`);
+  }
+
+  // Verify implementation
+  const tables = (await ethers.getContractFactory("TablelandTables")).attach(
+    proxy
+  );
+  const impl = await upgrades.erc1967.getImplementationAddress(tables.address);
+  await run("verify:verify", {
```
Comment (sanderpick) [inline scripts/verify.ts:21]: We can use this to verify the implementation contracts once this code base matches what is deployed.

**Comment by joewagner** [comment]
Nicely done!  Lot's of good changes here.  All works locally in regard to locking a table, and the util stuff is really handy.
The change on npm from @tableland/eth to @tableland/evm has some implications to npm.  I believe we want to deprerciate @tableland/eth with the instructions here: https://docs.npmjs.com/cli/v8/commands/npm-deprecate so that anyone trying to use @tableland/eth will get a warning.
We will also want to up date the @tableland/sdk dependency from eth -> evm

**Review by joewagner — APPROVED** [review]

**Pull request merged**